lang/de_DE.trn

```diff
@@ -17,59 +17,59 @@
 # See TRANSLATION.txt for more detailed information.
 #
 o0 Autobrake
-t0 automatisches Verlangsamen
+t0 Automatisches Verlangsamen
 o1 Swap Axes Looking Down
-t1 Achsen invertieren, wenn man nach unten schaut
+t1 Achsen beim Nach-unten-schauen umkehren
 o2 Camera Acceleration: 
 t2 Kamera-Beschleunigung: 
 o3 Camera Drag: 
-t3 Kamera ziehen: 
+t3 Kamerazug: 
 o4 Camera Max Speed: 
-t4 Max. Geschwindigkeit der Kamera: 
+t4 Kamera-Maximalgeschwindigkeit: 
 o5 Camera Braking Speed: 
-t5 Kamera Bremsgeschwindigkeit: 
+t5 Kamera-Bremsgeschwindigkeit: 
 o6 Mouse Speed: 
-t6 Maus-Geschwindigkeit: 
+t6 Mausgeschwindigkeit: 
 o7 Undo Limit: 
-t7 Rückgängig machen - Limit: 
+t7 Rückgangsbegrenzung: 
 o8 Invert Mouse
 t8 Maus invertieren
 o9 Low Detail Height
-t9 Low Detail - Anfangshöhe
+t9 Detailgrenze
 o10 Block Buffer (MB):
-t10 Block Puffer (MB):
+t10 Blockspeicher (MB):
 o11 Set Window Placement
 t11 Fensterposition speichern
 o12 Rotate block with brush
 t12 Block mit Pinsel drehen
 o13 Window Resize Alert
-t13 Fenstergrössen-Änderungs Warnung
+t13 Warnung beim Ändern der Fenstergröße
 o14 Long-Distance Mode
-t14 Fern-Modus
+t14 Fernmodus
 o15 Fly Mode
-t15 Flug-Modus
+t15 Flugmodus
 o16 Language
 t16 Sprache
 o17 Static Coords While Nudging
-t17 Statische Koordinaten während des Schiebens
+t17 Statische Koordinaten beim Schieben
 o18 Change Spawners While Nudging
-t18 Spawners ändern während des Schiebens
+t18 Spawner ändern beim Schieben
 o19 Change
 t19 Wechseln
 o20 Options
 t20 Optionen
 o21 OK
-t21 OK
+t21 Fertig
```

Comment:
Im Spiel wurde es auch mit "Fertig" übersetzt. In MCEdit passt es aber nicht überall, ist mir beim weiteren Übersetzen aufgefallen. In der nächsten Version ändere ich das wieder, keine Sorge :)